### ISSUE TITLE
Sequence Processing

### ISSUE THREAD

**Issue by Chatha-Sphere** [Chatha-Sphere/pno-ai#6]
Some of the required steps will include:

-one-hot encoding the action sequences
-padding sequences to a uniform length for batching
-offsetting sequences for input/output pair ("hello worl", "ello world")

Think about how to initialize the training vector (?) All zeroes?
Helpful link(s):
- https://blog.floydhub.com/a-beginners-guide-on-recurrent-neural-networks-with-pytorch/
- https://towardsdatascience.com/writing-like-shakespeare-with-machine-learning-in-pytorch-d77f851d910c

**Comment by Chatha-Sphere** [comment]
More thoughts: 
- Sequences should be no less than 32 characters and no greater than 512. 
- check out https://github.com/tensorflow/magenta/blob/master/magenta/music/encoder_decoder.py
for a way to "extend" input sequences based on the model.

**Comment by Chatha-Sphere** [comment]
https://stackoverflow.com/questions/49466894/how-to-correctly-give-inputs-to-embedding-lstm-and-linear-layers-in-pytorch/49473068#49473068
